tb_pulumi/cloudwatch.py

```diff
@@ -94,6 +106,60 @@ def __init__(
         )
 
 
+class LoadBalancerAlarmGroup(tb_pulumi.monitoring.AlarmGroup):
+    """In AWS, a load balancer can have a handful of types: ``application`` , ``gateway`` , or ``network`` . The metrics
+    emitted by the load balancer - and therefore the kinds of alarms we can build - depend on which type it is. However,
+    all types are represented by the same class, ``aws.lb.load_balancer.LoadBalancer`` . This necessitates a class for
+    disambiguation. The ``load_balancer_type`` is an Output, so here we wait until we can determine that type, then
```

Comment:
thanks for the clarifying documentation! this is helpful for new users like me to pick up context quickly